### ISSUE TITLE
DoubleUpDown isn't culture aware

### ISSUE THREAD

**Issue by buildYourOwnGolfSimulator** [Dirkster99/NumericUpDownLib#23]
The problem is if you have a cultureInfo which uses decimal comma instead of decimal point. The binding from Text to Value<double> will do a conversion which doesn't respect the CurrentCulture. As soon as the value will have any fraction in it, will the TextBox instead display the min value.
I solved this with adding a class
```
namespace NumericUpDownLib.Base
{
    public class CultureAwareBinding : Binding
    {
        public CultureAwareBinding()
        {
            ConverterCulture = CultureInfo.CurrentCulture;
        }
    }
}
```
and then also changed the binding for the TextBox
`Text="{local:CultureAwareBinding RelativeSource={RelativeSource TemplatedParent}, Path=Value}"
`
Would be great to get a new Nuget package with these fixes so I will not have to keep edited source...

**Comment by Dirkster99** [comment]
Hi, thanks for the feedback, would you please create a PR so I am not making an additional mistake when adopting your code?

PS: I was unaware of your problem because I always use English settings (even though I am German) but I will test your solution on different cultures for sure and create a new NuGet ASAP

**Comment by Dirkster99** [comment]
This is fixed in release of version 3.0
